spec/OBD/OBD.vspec

```diff
@@ -34,7 +47,7 @@
   description: Malfunction Indicator Light (MIL) False = Off, True = On
 
 - Status.DTCCount:
-  datatype: uint32
+  datatype: float
```

Comment:
This really is a "count" as in "number of errors. No "sensor", Wouldn't it make more sense to keep this as some sort of int?

Reply:
Yes we could have it as int. Then possibly uint8 as only seven bits are used in OBD (see https://en.wikipedia.org/wiki/OBD-II_PIDs#Service_01_PID_01)